plugins/sharding/stream_execute_context.rb

```diff
@@ -56,6 +58,23 @@ def need_look_ahead?
         true
       end
 
+      def parse_order(input, name)
+        order = input[name]
+        return :ascending if order.nil?
+
+        case order
+        when "ascending"
+          :ascending
+        when "descending"
+          :descending
+        else
+          message =
+            "[logical_range_filter] #{name} must be " +
```

Comment:
We should use `command_name` here.

Reply:
Sorry, I have overlooked.
I have fixed it.